src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -186,39 +186,41 @@ private static bool TryGetTextForOperator(SyntaxToken token, Document document,
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.ColonColonToken))
+            if (token.IsKind(SyntaxKind.InterpolatedStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringEndToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringTextToken))
             {
-                text = "::_CSharpKeyword";
+                text = "$_CSharpKeyword";
                 return true;
             }
 
-            if (token.Kind() == SyntaxKind.ColonToken && token.Parent is NameColonSyntax)
+            if (token.IsVerbatimStringLiteral())
             {
-                text = "cs_namedParameter";
+                text = "@_CSharpKeyword";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.QuestionToken) && token.Parent is ConditionalExpressionSyntax)
+            if (token.IsKind(SyntaxKind.ColonColonToken))
             {
-                text = "?_CSharpKeyword";
+                text = "::_CSharpKeyword";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.EqualsGreaterThanToken))
+            if (token.Kind() == SyntaxKind.ColonToken && token.Parent is NameColonSyntax)
             {
-                text = "=>_CSharpKeyword";
+                text = "cs_namedParameter";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.PlusEqualsToken))
+            if (token.IsKind(SyntaxKind.QuestionToken) && token.Parent is ConditionalExpressionSyntax)
```

Comment:
should we do this for `:` as well.

Reply:
Pressing F1 on `:` in the following snippet:

```
    public class Test
    {
    }

    public class Class1 :/*F1 here*/ Test
    {
    }
```

Redirects to https://docs.microsoft.com/en-us/dotnet/api/system.nullable-1 on Visual Studio 16.6.4 on my machine. Surely the link in very weird.

@CyrusNajmabadi Do you want me to try fix everything I can in the F1 area in this PR? or keep it only for the newly added ones for string interpolation and verbatim string?